lib/internal/Magento/Framework/Mview/View/Changelog.php

```diff
@@ -53,12 +60,14 @@ public function __construct(\Magento\Framework\App\ResourceConnection $resource)
      * Check DB connection
      *
      * @return void
-     * @throws \Exception
+     * @throws \Magento\Framework\Exception\SessionException
      */
     protected function checkConnection()
     {
         if (!$this->connection) {
-            throw new \Exception("The write connection to the database isn't available. Please try again later.");
+            throw new SessionException(
```

Comment:
I am not sure this is the best place for SessionException.
I think, \Magento\Framework\DB\Adapter\ConnectionException will be more suitable.